lib/wadler/print.rb

```diff
@@ -75,17 +75,28 @@ def group(indent = 0, open_obj = '', close_obj = '',
     end
 
     # @param indent [Integer] nest indentation
+    # @param open_obj [String] nest oppening delimiter
```

Comment:
Pun intended?

Reply:
I don't see the pun...

Reply:
nest **oppening** delimiter

Reply:
Oh, didn't even realize. It appears that the typo checker also didn't notice. Will fix this